modules/datastore/modules/datastore_mysql_import/src/Service/MysqlImport.php

```diff
@@ -134,7 +214,7 @@ private function getDatabaseConnectionCapableOfDataLoad() {
   private function generateTableHeaders(array $columns): array {
     return array_replace([], ...array_map(function ($column) {
       // Sanitize the supplied table header to generate a unique column name.
-      $header = $this->sanitizeHeader($column);
+      $header = $this->sanitizeHeader($column ?? '');
```

Comment:
This looks odd - could use a comment explaining?

Reply:
This has to do with some strange undocumented behavior I encountered with the `str_getcsv()` function in PHP.
Essentially, if an empty string is encountered, it returns an array with a single `NULL` element; in any other case, an empty string is used to represent empty headers.
This takes that odd behavior into account by null-coalescing any potentially `NULL` column names to an empty string.
I'll add a comment to the code too.

Reply:
I moved the check for `NULL` column names up to where the headers are first fetched and added a comment in an attempt to make things a bit more clear.